spokestack/extensions/webrtc/agc.pyx

```diff
@@ -0,0 +1,94 @@
+"""
+This module contains the AutomaticGainControl class implemented in Cython
+"""
+# distutils: sources = filter_audio/agc/analog_agc.c, filter_audio/agc/digital_agc.c
+# distutils: include_dirs = filter_audio/agc/include/
+cimport numpy as np
+cimport cagc
+
+
+MIC_MAX = 255
+MIC_TARGET = 180
+
+np.import_array()
+
+cdef class AutomaticGainControl:
+    cdef void*_agc
+    cdef int _sample_rate
```

Comment:
The only member variable that is actually used outside the constructor is the `_agc` member. The rest can be removed.